src/components/Questionnaire/ValueSetSearchContent.tsx

```diff
@@ -387,7 +390,7 @@ export default function ValueSetSearchContent({
         variant="destructive"
         disabled={clearFavouritesMutation.isPending}
         onConfirm={() => {
-          clearFavouritesMutation.mutate({});
+          clearFavouritesMutation.mutate(undefined);
```

Comment:
```suggestion
          clearFavouritesMutation.mutate();
```

ps: I'm sorry I know it's in work in progress state but this thing just got my eyes